easy_transformer/evals.py

```diff
@@ -3,12 +3,14 @@
 A file with some rough evals for models - I expect you to be likely better off using the HuggingFace evaluate library if you want to do anything properly, but this is here if you want it and want to eg cheaply and roughly compare models you've trained to baselines.
 """
 
+import einops
```

Comment:
What's the algorithm for refactoring imports?

Reply:
[isort](https://github.com/PyCQA/isort)

> isort is a Python utility / library to sort imports alphabetically, and automatically separated into sections and by type.